pages/modules/[module].tsx

```diff
@@ -198,59 +198,101 @@ const ModulePage: NextPage<ModulePageProps> = ({
                     </button>
                   )}
                 </div>
-                <h2 className="text-2xl font-bold mt-4">
-                  Dependencies{' '}
-                  <small className="text-sm font-normal text-gray-500">
-                    (of version {selectedVersion})
-                  </small>
-                </h2>
-                <div>
-                  <ul className="mt-4">
-                    {versionInfo.moduleInfo.dependencies.map((dependency) => (
-                      <Link
-                        key={dependency.module}
-                        href={`/modules/${dependency.module}/${dependency.version}`}
-                      >
-                        <li className="border rounded p-2 mt-2 flex items-center gap-4 hover:border-gray-800">
-                          <div className="rounded-full border h-14 w-14 grid place-items-center">
-                            {dependency.version}
-                          </div>
-                          <div>{dependency.module}</div>
-                        </li>
-                      </Link>
-                    ))}
-                    {versionInfo.moduleInfo.dependencies.length === 0 && (
-                      <span>No dependencies</span>
-                    )}
-                  </ul>
+                <div className="mt-4">
+                  <h2 className="text-2xl font-bold mt-4">Dependency graph</h2>
                 </div>
-                <h2 className="text-2xl font-bold mt-4">Dependents</h2>
-                <div>
-                  <ul className="mt-4">
-                    {shownReverseDependencies.map((revDependency) => (
-                      <Link
-                        key={revDependency}
-                        href={`/modules/${revDependency}`}
+                <div className="mt-4">
+                  <details open>
+                    <summary>
+                      <span role="heading" aria-level={2} className="text-1xl mt-4">
```

Comment:
`h2` is now a `span` to avoid needing to style the headings; `aria-level`/`role` is used to preserve semantics